.github/workflows/ci.yml

```diff
@@ -5,10 +5,15 @@
 # This workflow will install a prebuilt Ruby version, install dependencies, and
 # run tests and linters.
 name: "CI"
+
 on:
   push:
-    branches: [ main, deploy/** ]
```

Comment:
The deploy/** is used for the Pulumi setup. It's useful to have another branch you can push to to deploy on a fork.

Reply:
ok.  I removed it because we didn't have any branches that matched that.

While i'm not convinced we will want to use this, i put it back.  I'm assuming we are going to want:
- auto push to staging on main
- auto push to prod on release

or something similar.  Either way, i put this back for now.